data/common.yaml

```diff
@@ -20,11 +20,15 @@ lookup_options:
       strategy: unique
       # Disable this for now.
       #knockout_prefix: --
-
-# This is all that we support on unknown operating systems
+simp::vardir_owner: 'root'
+simp::vardir_group: 'root'
+simp::vardir_mode: '0750'
 simp::scenario_map:
   none: []
-
+  remote_access: []
```

Comment:
Not sure I agree with adding these. If the user's expectation is that these should actually *do* something (and it should be), this will not meet their expectation.

This is particularly true of other Linux-type OSs and may give a false sense of compliance.

Could we change these to including a class that just adds a `notify` statement that you're using an unsupported OS with whatever scenario you've chosen so it isn't a silent failure?

Reply:
I had a notify added in an older PR to check the result of the union, if the classlist was empty it sent a notify telling you that 'include simp' was not going to include classes that could enforce anything

Reply:
that also would cover the case where you are running a supported system but decided to disable everything

Reply:
@heliocentric The `notify` would be fine as long as we can allow the user to disable it (apparently, they don't want lots of hosts yelling about things they already know).